docs/developer.md

```diff
@@ -48,9 +48,9 @@ As of now the example config enables OAuth2 based authentication. It can be chan
 
 OAuth2 based authentication requires GitHub application, this can be created by following this 
 [tutorial](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
-Please set the `clientid` and the `clientsecret` in the auth section, with the GitHub generated values.
+Please set the `token` in the github section and the `clientid` and the `clientsecret` in the auth section, with the GitHub generated values.
```

Comment:
There's no reference to any documentation explaining where this token comes from.

Reply:
I added a short section about it to the `github-app.md` where I feel the description belongs to. Let me know if it makes sense.